src/item.cpp

```diff
@@ -1614,13 +1614,7 @@ std::string item::info(bool showtext, std::vector<iteminfo> &dump_ref) const
                     known_recipes.push_back(r);
                 }
             }
-            if (known_recipes.size() > 24) {
-                dump->push_back(iteminfo("DESCRIPTION", "--"));
-                dump->push_back(iteminfo("DESCRIPTION", _("You know dozens of things you could craft with it.")));
-            } else if (known_recipes.size() > 12) {
-                dump->push_back(iteminfo("DESCRIPTION", "--"));
-                dump->push_back(iteminfo("DESCRIPTION", _("You could use it to craft various other things.")));
-            } else {
+            if (OPTIONS["ITEM_SHOW_ALL_RECIPES"] || known_recipes.size() <= 12) {
```

Comment:
You could instead make the option an int and replace that 12 here with the option.